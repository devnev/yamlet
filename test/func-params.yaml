locals:
  f: !func
    params:
    - x
    - y
    result: (x + y)
  z: 3
result:
  val: (f(1, 2))
  val2: (f(z, 1))
