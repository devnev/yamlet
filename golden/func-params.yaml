val: 3
val2: 4
